ecosystem/op-deployer.md

```diff
@@ -0,0 +1,163 @@
+# Purpose
+
+Introduce `op-deployer`, an end-to-end chain deployment tool, and get buy-in on its architecture and implementation.
+
+## Goals
+
+- Provide a canonical way to deploy chains.
+- Enable deploying multiple L2s to a single L1 as part of a single logical Superchain.
+- Make it easier for teams like Protocol and Proofs to maintain their deployment tooling.
+
+## Non-Goals
+
+- Replacing `op-e2e`. This will be discussed later, once `op-deployer` is live.
+
+# Problem Statement + Context
+
+As we've
+discussed [previously](https://docs.google.com/document/d/13f8UoO9j05PJdvAZDWuVJWD1UAIEAfb8VbphBnPE3cA/edit#heading=h.jxalqw91xhd0),
+deploying and upgrading OP Chains remains a complicated and frustrating process. Users are directed to a tutorial
+with [18 manual steps](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup) to follow in order
+to create their own testnet. Within Labs, we use a combination of different tools depending on where the chain is to be
+hosted:
```

Comment:
I believe the superchain-ops-team uses some variant of the deploy-scripts, with some instrumentation for Safe contracts / deploy-tx reliability. I'm not entirely familiar with it, but I think it should be in this list.

Reply:
Ok, I'll track this down.